python/cudf/bindings/csv.pyx

```diff
@@ -249,6 +249,7 @@ cpdef cpp_write_csv(
     false_value = 'False'.encode()
     csv_writer.false_value = false_value
     csv_writer.include_header = header
+    csv_writer.chunk_rows = 10240000
```

Comment:
This should be a parameter at the Cython and Python API levels and then set this based on the parameter.

Reply:
This will be done as part of #2137